java/arcs/core/storage/BackingStore.kt

```diff
@@ -39,7 +39,14 @@ class BackingStore<Data : CrdtData, Op : CrdtOperation, T>(
 ) {
     private val storeMutex = Mutex()
     // TODO(b/158262634): Make this CacheMap Weak.
-    /* internal */ val stores = LruCacheMap<String, StoreRecord<Data, Op, T>>()
+    /* internal */ val stores = LruCacheMap<String, StoreRecord<Data, Op, T>>(
+        50,
```

Comment:
How did we determine 50 and when might we need/want to change it, would we need to change other things along with it? Worth a comment in the code IMO.